metricbeat/module/openmetrics/collector/_meta/data.json

```diff
@@ -10,15 +10,18 @@
         "period": 10000
     },
     "openmetrics": {
+        "help": "Total number of connections closed that were made to the listener of a given name.",
         "labels": {
-            "job": "openmetrics"
+            "job": "openmetrics",
+            "listener_name": "http"
         },
         "metrics": {
-            "up": 1
-        }
+            "net_conntrack_listener_conn_closed_total": 0
```

Comment:
Was this included by accident? Everything else in the PR looks like natural in-place substitutions, but this seems unrelated. (If the `data.json` changes weren't here I would approve it.)

Reply:
sorry - I must have added them by accident and will remove them.

Reply:
file got removed from the PR.